EFWorkshop/EFWorkshopCodeFirst/Program.cs

```diff
@@ -0,0 +1,24 @@
+﻿using EFWorkshopCodeFirst;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace EfWorkshop
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            var host = CreateHostBuilder(args).Build();
+
+            var appRunner = ActivatorUtilities.CreateInstance<AppRunner>(host.Services);
```

Comment:
We don't really need app hosting here, since you are not using dependency injection and there is no `appsettings.json` file in this project. Just put all the logic from `AppRunner` into here. Before doing so, please remove everything from `Program` leaving just empty `Main`.